app/assets/stylesheets/header-footer-only.scss

```diff
@@ -1,12 +1,12 @@
+@import "govuk_publishing_components/govuk_frontend_support";
+
```

Comment:
Sorry I'm not clear on the difference, do we need to import `@import 'govuk_publishing_components/component_support';` too?

Reply:
The component support stuff is if you're pulling in component sass - it's the mixins and variables for the components. It shouldn't be needed (the cookie banner doesn't use anything from it) but in the light of your comment about the feedback component, I might need to rethink it. Will have a look.

Reply:
Yes, we do now. I've added it. It's not ideal. I originally split them into the two imports (component and govuk_frontend) because some of it adds CSS weight (specifically the branding colours) and static only needs govuk-frontend (not the branding, which would be duplicated in apps as well).

With this development we might as well only have the one support file. Rather than going back to the gem and trying to come up with a solution I think we should go ahead with this addition and I'll try to work up an alternative later. Sorry, slight tech debt introduction here.